terraform/060-simplesamlphp/README.md

```diff
@@ -24,6 +24,7 @@ This module is used to create an ECS service running simpleSAMLphp.
  - `id_broker_assert_valid_ip` - Whether or not to assert valid ip for calling id-broker
  - `id_broker_trusted_ip_ranges` - List of trusted ip blocks for ID Broker
  - `id_broker_base_uri` - Base URL to id-broker API
+ - `mfa_setup_url` - URL to setup MFA
```

Comment:
It may be worth waiting until a subsequent release to add it here, but we are nearly done adding support for an `MFA_LEARN_MORE_URL` env. var.

Reply:
I went ahead and just added the learn more stuff in now too.